css/validation.css

```diff
@@ -2,35 +2,37 @@
    Message Area
    ====================================================================== */
 .message-area {
-  /* Message area to provide feedback used to display messages such as...
-     "Sorry, that email or password is not recognised." or "No comments yet :("
-  */
-  font-size: 1em;
-  color: #31708f;
-  background-color: #d9edf7;
-  border: 1px solid #bce8f1;
-  border-radius: 4px;
-  padding: 10px 20px;
-  margin: 0 0 20px 0;
-  text-align: left;
-  cursor: pointer;
+    /* Message area to provide feedback used to display messages such as...
+       "Sorry, that email or password is not recognised." or "No comments yet :("
+       */
+    font-size: 1em;
+    color: #31708f;
+    background-color: #d9edf7;
+    border: 1px solid #bce8f1;
+    border-radius: 4px;
+    padding: 10px 20px;
+    margin: 0 0 20px 0;
+    text-align: left;
+    cursor: pointer;
+    position: relative;
 }
 
 .message-area > .icon--cross {
-  float: right;
-  width: 10px;
-  margin-top: 5px;
-  content: url("../images/cross.svg");
+    position: absolute;
+    width: 10px;
+    top: calc(50% - 5px);
+    left: calc(98% - 10px);
```

Comment:
`calc()` isn't supported in IE8 and earlier. I had thought of decent fallbacks and attempted to test them but it looks like the close icon doesn't appear in IE at all (even the latest version). The close icon not appearing in IE likely isn't a huge problem because it doesn't effect the functionality; it simply served as a visual aid to let the user know that the message could be dismissed. The message can still be dismissed by simply clicking anywhere on the message box.